data-prepper-api/src/main/java/com/amazon/dataprepper/model/trace/JacksonSpan.java

```diff
@@ -56,8 +56,6 @@ protected JacksonSpan(final Builder builder) {
         super(builder);
 
         checkArgument(this.getMetadata().getEventType().equals("TRACE"), "eventType must be of type Trace");
```

Comment:
nit: unrelated to this PR, "TRACE" can be moved to static constant for maintainability reasons.